cypress/support/commands.js

```diff
@@ -873,7 +873,7 @@ Cypress.Commands.add('uploadSubmissionFiles', (files, options) => {
 Cypress.Commands.add('changeLanguage', (language, contextPath) => {
 	contextPath = contextPath || 'publicknowledge';
 
-	cy.get('.app__userNav > button').click();
-	cy.get('.app__userNav a:contains("français")').click();
+	cy.get('[data-cy="app-user-nav"] > button').click();
+	cy.get('[data-cy="app-user-nav"] a:contains("français")').click();
 	cy.wait(2000);
```

Comment:
right.. I need to delete this file :-D..  keep forgetting..  only commands_new_workflow.js should be used I think  But thats not to worry about in this PR